iroha/src/wsv.rs

```diff
@@ -14,10 +12,10 @@ use crate::prelude::*;
 pub struct WorldStateView {
     /// The world - contains `domains`, `triggers`, etc..
     pub world: World,
-    /// Hashes of the committed and rejected transactions.
-    pub transactions: HashMap<Hash, TransactionValue>,
```

Comment:
I mean we should leave HashSet here. To quickly check if transaction is not in blockchain, hashes are a lot smaller than actual txs.